src/Products/sources/LocalStorageSource.js

```diff
@@ -1,4 +1,11 @@
+import testData from "./test-data.json";
```

Comment:
you don't need this import in this file, do you?

Reply:
I am using `testData` to insert some data on Local Storage to have something to assert in the tests.